bugbot/rules/workflow/no_severity_ni.py

```diff
@@ -137,56 +134,27 @@ def get_bz_params(self, date):
             "f5": "product",
             "o5": "changedafter",
             "v5": lookup,
-            "f6": "product",
+            "n6": 1,
+            "f6": "component",
             "o6": "changedafter",
-            "v6": lookup_nag,
-            "n7": 1,
-            "f7": "component",
-            "o7": "changedafter",
+            "v6": lookup,
+            "f7": "creation_ts",
+            "o7": "lessthaneq",
             "v7": lookup,
             "f8": "CP",
-            "j9": "AND",
+            "j9": "OR",
             "f9": "OP",
-            "n10": 1,
-            "f10": "component",
-            "o10": "changedafter",
-            "v10": lookup,
-            "f11": "component",
-            "o11": "changedafter",
-            "v11": lookup_nag,
-            "n12": 1,
-            "f12": "product",
-            "o12": "changedafter",
-            "v12": lookup,
+            "f10": "bug_severity",
+            "o10": "changedfrom",
+            "v10": "critical",
+            "f11": "bug_severity",
+            "o11": "changedfrom",
+            "v11": "major",
+            "f12": "bug_severity",
+            "o12": "changedfrom",
+            "v12": "blocker",
```

Comment:
Are these still necessary?

Reply:
Oh, it looks like we no longer have any, so we can remove this.